org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/repositories/AbapGitView.java

```diff
@@ -127,6 +139,11 @@ public void selectionChanged(IWorkbenchPart part, ISelection selection) {
 	public void init(IViewSite site) throws PartInitException {
 		super.init(site);
 
+// Create a new abapGit service when AbapGit Repositories View is opened.
+		if (this.abapGitService == null) {
+			this.abapGitService = new AbapGitService();
```

Comment:
You can also use the service factory `AbapGitUIServiceFactory.createAbapGitService()` for creating service instances.